spec/index.bs

```diff
@@ -13493,8 +13493,8 @@ specified points.
 
 The <dfn dfn>supported context formats</dfn> are a [=set=] of {{GPUTextureFormat}}s that must be
 supported when specified as a {{GPUCanvasConfiguration}}.{{GPUCanvasConfiguration/format}}
-regardless of the given {{GPUCanvasConfiguration}}.{{GPUCanvasConfiguration/device}},
-initially set to: &laquo;{{GPUTextureFormat/"bgra8unorm"}}, {{GPUTextureFormat/"rgba8unorm"}},
+regardless of the given {{GPUCanvasConfiguration}}.{{GPUCanvasConfiguration/device}}, set to:
+&laquo;{{GPUTextureFormat/"bgra8unorm"}}, {{GPUTextureFormat/"rgba8unorm"}},
 {{GPUTextureFormat/"rgba16float"}}&raquo;.
```

Comment:
"set" still makes it sound like a variable, to me, though it's definitely better. I think the problem here is we did intend for it to be variable (constant over time, but possibly different per-device-per-canvas or something like that) - but we made the variable global instead of actually putting on an object. But we can cross that bridge when we get to it; for now what about something like:

> The `supported context formats` are the set: &laquo; ... &raquo;.
> These formats must be supported...

Reply:
Good suggestion! Updated.